bin/local-env/install-wordpress.sh

```diff
@@ -219,3 +228,11 @@ wp option patch insert web_stories_experiments enableSVG 0
 wp user list --format=yaml
 wp post list --post_type=attachment --format=yaml
 wp plugin list --format=yaml
+
+# Only install woocommerce on latest version of WordPress.
+if [ "$WP_VERSION" == "latest" ]; then
+	echo -e $(status_message "Import sample woocommerce products...")
+	wp import /var/www/html/wp-content/plugins/woocommerce/sample-data/sample_products.xml --authors=skip --quiet
+	# deactivate test etc... can activate as needed
+	wp plugin deactivate woocommerce 
```

Comment:
Does the plugin need to be activated further above in the first place? Is it required for the import to work?

Otherwise we can remove this line & the `--activate` flag above.

Reply:
Does the plugin need to be activated further above in the first place?

```bash
> STATUS: Import sample woocommerce products...
Failed to import &#8220;V-Neck T-Shirt&#8221;: Invalid post type product<br />Failed to import 
```
☝️ 

Without activating above the import fails